ang/civicase/activity/actions/services/move-copy-activity-action.service.js

```diff
@@ -69,19 +69,7 @@
      * @returns {object} api parameters for Case.getlist
      */
     function getCaseListApiParams () {
-      var firstCaseTypeCategoryId =
-        _.keys(CaseTypeCategory.getCategoriesWithAccessToBasicCaseInformation())[0];
-      var allCaseTypeCategoryNames =
-        _.values(CaseTypeCategory.getCategoriesWithAccessToBasicCaseInformation());
-
-      return {
-        'case_type_id.case_type_category': firstCaseTypeCategoryId,
-        params: {
-          'case_id.case_type_id.case_type_category': {
-            IN: allCaseTypeCategoryNames
-          }
-        }
-      };
+      return {};
```

Comment:
I think this change might break some tests. Also, I think we don't need the `getCategoriesWithAccessToBasicCaseInformation` function anymore, do we @deb1990 ?